### PR TITLE
Option to display last changed in glance-card

### LINKED ISSUE #110: Show last changed instead of state in glance cards
I would like to override the `state` value displayed on glance cards.

Instead of `detected`/`clear`, the default for `class: motion` sensors, I would like to use a template to instead show the number of minutes since the last motion was detected. So instead of "clear", it would say "5 mins ago" if motion was detected 5 mins ago.

screenshot 2018-07-18 15 33 16

I would like something like this:

```yaml
cards:
  - type: glance
    title: Motion Sensors
    entities:
      - entity: binary_sensor.kitchen_motion
        name: Top Floor
        # Instead of just clear/detected, show last activation time:
        value_template: {{ states.binary_sensor.kitchen_motion.last_changed | timeago }}
```

### PR DESCRIPTION
**Pull request by iantrich**
Closes https://github.com/home-assistant/ui-schema/issues/110

### CODE REVIEW

**Comment by bramkragten** [comment]
Shouldn't this be an option on the entity instead of the card?

**Comment by iantrich** [comment]
That's the direction I started but all such options are card level. Was going to bring that up outside of this.

**Comment by iantrich** [comment]
@balloob @thomasloven input on moving `show` options of glance card to entity-level?
i.e.
`show_icon`
`show_name`
`show_state`
`show_last_changed`

**Comment by bramkragten** [comment]
I can understand that `show_icon`, `show_name`, `show_state` are on card level as they change the layout. But `show_last_changed` doesn't, it affects what data is shown.

**Comment by iantrich** [comment]
hmm, fair point

File: src/panels/lovelace/cards/hui-glance-card.ts [hunk 1]
```diff
@@ -210,7 +210,8 @@ export class HuiGlanceCard extends LitElement implements LovelaceCard {
                 ${computeStateDisplay(
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-glance-card.ts]: We should not add this to `computeStateDisplay`. Instead, make it a turnary:

```ts
entityConf.show_last_changged ? renderShowLastChanged(…) : computeStateDisplay(…)
```

File: src/panels/lovelace/cards/types.ts [hunk 2]
```diff
@@ -80,6 +80,7 @@ export interface GaugeCardConfig extends LovelaceCardConfig {
 export interface ConfigEntity extends EntityConfig {
   tap_action?: ActionConfig;
   hold_action?: ActionConfig;
+  show_last_changed?: boolean;
```
Comment (balloob) [inline src/panels/lovelace/cards/types.ts]: This should not be a generic option, this should only be added to the glance entity type.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**